scripts/SignImage

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# IBM_PROLOG_BEGIN_TAG
+# This is an automatically generated prolog.
+#
+# $Source: scripts/SignImage $
+#
+# OpenPOWER sb-signing-utils Project
+#
+# Contributors Listed Below - COPYRIGHT 2016
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# IBM_PROLOG_END_TAG
+
+# the container consists of 3 Hardware Keys 
+#      * HW-KeyA - Hardware Public Key A
+#      * HW-KeyB - Hardware Public Key B
+#      * HW-KeyC - Hardware Public Key C
+# and 3 corresponding Hardware Signatures 
+#      * HW-SigA - Hardware Signature A
+#      * HW-SigB - Hardware Signature B
+#      * HW-SigC - Hardware Signature C
+# It also consists of upto 3 Software Keys (depends on project settings)
+#      * SW-KeyP - Software Public Key P
+#      * SW-KeyQ - Software Public Key Q
+#      * SW-KeyR - Software Public Key R
+# and upto 3 corresponding Software Signatures 
+#      * SW-SigP - Software Signature P
+#      * SW-SigQ - Software Signature Q
+#      * SW-SigR - Software Signature R
+ 
+SIGNTOOL=./signtool
+
+IMG_FILE=./images/skiboot.bin
```

Comment:
I'm unsure what the purpose of this tool is, but should all of these be ?= set?  We also support XZ compressed skiboot images which are named skiboot.bin.xz.